pkgs/applications/version-management/git/default.nix

```diff
@@ -84,7 +85,7 @@ stdenv.mkDerivation (finalAttrs: {
     done
   '';
 
-  nativeBuildInputs = [ gettext perlPackages.perl makeWrapper pkg-config ]
+  nativeBuildInputs = [ deterministic-host-uname gettext perlPackages.perl makeWrapper pkg-config ]
```

Comment:
I can also make this optional when `buildPlatform` and `hostPlatform` differ if that's preferred.

Reply:
It’s better for native and cross builds to be the same whenever possible, so it should be fine as‐is.